### ISSUE TITLE
Safari bugs

### ISSUE THREAD

**Issue by TrySound** [webstudio-is/webstudio-builder#1818]
- canvas resize does not work
- space and inputs scrub does not work
- css preview is not scrollable
- tooltip list of properties is not scrollable

**Comment by johnsicili** [comment]
Tested on Safari 18

No longer an issue:
- css preview is not scrollable

Still issues:
- canvas resize does not work
- space and inputs scrub does not work

Unsure what this one means:
- tooltip list of properties is not scrollable

**Comment by TrySound** [comment]
List of properties in tooltip is no longer scroll area, so not an issue

**Comment by johnsicili** [comment]
Scrub now works in Safari except on spacing inputs. But if you open the spacing input it allows scrub.

Canvas resizing also works.

Though in both cases, as you are scrubbing/resizing, the cursor is highlighting what it's dragging over. That seems to be the main blocker but who knows... I'd need to spend some time building to see what other stuff pops up. Seems close though 🤷 

https://github.com/user-attachments/assets/e7e3ef9f-ba70-48e4-b08c-5eba9fb2e1c6

**Comment by istarkov** [comment]
<img width="284" alt="image" src="https://github.com/user-attachments/assets/e2e7dd9f-49e4-4650-adc8-9f709cc1cd65" />
Can't reproduce `Style source autocomplete has some extra text partially shown`

**Comment by o-be-one** [comment]
Had an issue with copy yesterday on Safari. But using cut worked so I cut, CMD+Z and then I paste where I need.

**Comment by o-be-one** [comment]
Firefox issue (latest MacOS):
It's not possible to rename component from the navigator. If you attend to push "delete" it will remove the component, so you revert with CMD+Z and it comes back like this in the screenshot. To go back to normal you have to double click another component from the navigator and things will come back as expected.
![Image](https://github.com/user-attachments/assets/35b9d6c7-9512-4a95-8c1e-5c3e70863c40)

Will continue to report visual issues here :).

**Comment by NatElkins** [comment]
I see folks are reporting issue with Firefox in this thread, but I don't even see how I can run the app in Firefox (at least, the deployed version).  I see this modal, with no way to dismiss it:

<img width="503" alt="Image" src="https://github.com/user-attachments/assets/76382116-a4e9-4b94-9905-5715623b53ec" />

In my ideal flow, I'd like to be able to "accept the risk" and deal with a subpar experience rather than have to switch browsers.  Is that something that could be considered?  Would help with beta testing as well.

**Comment by kof** [comment]
@NatElkins are you also ready to accept potential bugs? like things not working in the builder? 

We are considering making this thing dismissable.

**Comment by NatElkins** [comment]
Yes, definitely.  Something is better than nothing, and at least if I can encounter bugs, then perhaps I can fix them.

**Comment by kof** [comment]
What do you mean by disclaimer? Is this warning initially you need to click  away not enough?

**Comment by o-be-one** [comment]
Is it also possible to have a "feedback" button that would maybe redirect here?

**Comment by TrySound** [comment]
FYI https://github.com/webstudio-is/webstudio/pull/5158

**Comment by kof** [comment]
@onlineapps-cloud try now

**Comment by kof** [comment]
- [ ] Drag & drop in navigator in Firefox is not working

**Comment by gursuj** [comment]
Hello. On Firefox, I'm not able to use `clamp`. For example, try setting `font-size` to `clamp(1rem, 2.5vw, 2rem)`. This works fine on Chrome. Strange since Firefox supports clamp.
